install/antennas.csv

```diff
@@ -328,7 +328,7 @@ Trimble Navigation Ltd.,TRM57971.00,4127561886,RGHR,0.002,0,0,0,2016-08-15T23:43
 Trimble Navigation Ltd.,TRM57971.00,4127561893,VGOB,0.055,0,0,0,2016-05-12T01:01:58Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561896,PAWA,0.055,0,0,0,2016-03-01T02:36:52Z,9999-01-01T00:00:00Z
 Trimble Navigation Ltd.,TRM57971.00,4127561900,LRR1,0.002,0,0,0,2016-11-15T04:46:08Z,2017-05-22T00:14:00Z
-Trimble Navigation Ltd.,TRM57971.00,4127561900,GLOK,0.002,0,0,0,2017-06-12T04:25:09Z,9999-01-01T00:00:00Z
+Trimble Navigation Ltd.,TRM57971.00,4127561900,GLOK,0.002,0,0,0,2017-06-12T04:25:09Z,2017-09-20T03:24:31Z
```

Comment:
Hi @ozym , this is a wrong entry on the db that we are going to fix this morning. 
(all latest entries related to LOK1-GLOK will need to be corrected)